src/RazorConsole.Core/Vdom/Translators/HtmlListElementTranslator.cs

```diff
@@ -26,6 +26,11 @@ public bool TryTranslate(VNode node, TranslationContext context, out IRenderable
         }
 
         var isOrdered = tagName == "ol";
+        int offset = 0;
```

Comment:
Maybe move the overriding offset in `ol` to a separate PR, I create a tracking issue here: #115 

Also, in html the offset is configured by `start` argument, so let's respect that :)